[BUG] @shopware/unocss-design-tokens-layer is missing from releases

### Problem descriprion and current behaviour

The current [Vue Starter Template Extended](https://frontends.shopware.com/getting-started/templates/vue-starter-template-extended.html) tutorial fails, because `@shopware/unocss-design-tokens-layer` is missing from the npm registry. It fails during `pnpm i`:

```
 ERR_PNPM_FETCH_404  GET https://registry.npmjs.org/@shopware%2Funocss-design-tokens-layer: Not Found - 404
```

The [Vue Starter Template](https://frontends.shopware.com/getting-started/templates/vue-starter-template.html) tutorial fails also:

```
npm i && npm run dev
> cloned shopware/frontends#HEAD to my-store
npm error code EUNSUPPORTEDPROTOCOL
npm error Unsupported URL Type "workspace:": workspace:*
```

### Expected Behavior

No error during installation.

### Steps To Reproduce

Follow the tutorials.


### Anything else?

_No response_

--- a/.changeset/fine-laws-rush.md
+++ b/.changeset/fine-laws-rush.md
@@ -1,0 +1,5 @@
+---
+"@shopware/unocss-design-tokens-layer": major
+---
+
+Package release

--- a/packages/unocss-design-tokens-layer/package.json
+++ b/packages/unocss-design-tokens-layer/package.json
@@ -15,18 +15,8 @@
   "license": "MIT",
   "type": "module",
   "main": "./nuxt.config.ts",
-  "files": [
-    "dist",
-    "app",
-    "nuxt.config.ts",
-    "index.cjs",
-    "index.d.ts",
-    "app.config.ts",
-    "uno.config.ts"
-  ],
+  "files": ["app", "nuxt.config.ts", "index.d.ts", "uno.config.ts"],
   "scripts": {
-    "build": "nuxt prepare && unbuild",
-    "dev": "unbuild --stub",
     "lint": "biome check .",
     "lint:fix": "biome check . --write && pnpm run typecheck",
     "typecheck": "pnpm nuxt prepare && tsgo --noEmit"

--- a/templates/vue-starter-template-extended/package.json
+++ b/templates/vue-starter-template-extended/package.json
@@ -33,8 +33,8 @@
   "devDependencies": {
     "@biomejs/biome": "1.8.3",
     "@shopware/api-gen": "canary",
-    "@shopware/cms-base-layer": "workspace:*",
-    "@shopware/unocss-design-tokens-layer": "workspace:*",
+    "@shopware/cms-base-layer": "canary",
+    "@shopware/unocss-design-tokens-layer": "canary",
     "@unocss/nuxt": "66.6.0",
     "@unocss/core": "66.6.0",
     "typescript": "5.9.3",

--- a/templates/vue-starter-template/package.json
+++ b/templates/vue-starter-template/package.json
@@ -35,7 +35,7 @@
     "@shopware/composables": "canary",
     "@shopware/helpers": "canary",
     "@shopware/nuxt-module": "canary",
-    "@shopware/unocss-design-tokens-layer": "workspace:*",
+    "@shopware/unocss-design-tokens-layer": "canary",
     "@unocss/nuxt": "66.6.0",
     "@unocss/core": "66.6.0",
     "@vueuse/core": "14.1.0",

